Client/Auth/index.ts

```diff
@@ -144,6 +144,7 @@ export class Auth {
 		this.roles = undefined
 		this.features = undefined
 		window.sessionStorage.removeItem("authData")
+		window.sessionStorage.removeItem("cookie")
```

Comment:
you wanna keep that, its not much of a 2fa cooldown if it gets reset between sessions